runelite-client/src/main/java/net/runelite/client/plugins/screenshot/ScreenshotInput.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) 2018, Seth <http://github.com/sethtroll>
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice, this
+ *    list of conditions and the following disclaimer.
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+ * ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR
+ * ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ * LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+ * ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package net.runelite.client.plugins.screenshot;
+
+
+import java.awt.event.KeyEvent;
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.Locale;
+import javax.inject.Inject;
+
+import net.runelite.client.input.KeyListener;
+
+public class ScreenshotInput implements KeyListener
+{
+
+	private static final DateFormat TIME_FORMAT = new SimpleDateFormat("yyyy-MM-dd_HH-mm-ss", Locale.US);
+
+	private final ScreenshotConfig config;
+
+	@Inject
+	ScreenshotInput(ScreenshotConfig config)
+	{
+		this.config = config;
+	}
+
+	@Inject
+	private ScreenshotPlugin plugin;
+
+
+	@Override
+	public void keyPressed(KeyEvent event)
+	{
+	}
+
+	@Override
+	public void keyTyped(KeyEvent event)
+	{
+	}
+
+	@Override
+	public void keyReleased(KeyEvent event)
+	{
+		if (config.enableShortcut())
+		{
+			if (event.getKeyCode() == KeyEvent.VK_INSERT)
```

Comment:
Should be configurable key

Reply:
No other plugin has configurable keys (e.g the GE Hotkey lookup is set to Alt+Click ) - is it not easier and less error-prone to give a set shortcut key? In any case, how would this be implemented, some sort of input box? A dropdown list of reasonable (F11, Insert, Pause, etc) choices?

Reply:
This is one of the reasons we were avoiding adding more shortcuts. We need system that will make it simple, and add it to every shortcut we use.

Reply:
Understandable - what's the plan until then? Put this PR (and related PR's) on hold?

Reply:
Most likely not. But, we need to do it at some point. But for now, not adding the configurable option for it is correct.